src/components/ui/Suggest.jsx

```diff
@@ -83,6 +83,13 @@ const Suggest = ({
       fetchItems(value);
       setIsOpen(true);
       setLastQuery(value);
+      const topBarHandle = document.querySelector('.top_bar');
```

Comment:
This `<Suggest>` component is also used by the direction feature, where this effect shouldn't happen.
And in React you should avoid direct DOM manipulations like this, to prevent side effects breaking the rendering cycle.

Both these points show this should live elsewhere. As it looks like the things we already do in the `initTopBar` method of `PanelManager`, this is probably where this should go.